client/relay-chain-minimal-node/src/lib.rs

```diff
@@ -152,8 +158,10 @@ async fn new_minimal_relay_chain(
 		.extend(peer_sets_info(is_authority, &peer_set_protocol_names));
 
 	let request_protocol_names = ReqProtocolNames::new(genesis_hash, config.chain_spec.fork_id());
-	let (collation_req_receiver, available_data_req_receiver, pov_req_receiver, chunk_req_receiver) =
+	let (collation_req_receiver, available_data_req_receiver, chunk_req_receiver) =
 		build_request_response_protocol_receivers(&request_protocol_names, &mut config);
+	drain_unwanted_request_channels(&task_manager, chunk_req_receiver);
```

Comment:
Okay, this is unneeded. We can not do two things. Either we drop this channel or we do not even register the channel.